packages/grid/src/Grid.tsx

```diff
@@ -1739,7 +1720,7 @@ class Grid extends PureComponent<GridProps, GridState> {
   }
 
   handleWheel(event: WheelEvent): void {
-    this.notifyMouseHandlers('onWheel', event);
+    this.notifyMouseHandlers('onWheel', (event as unknown) as React.WheelEvent);
```

Comment:
`React.WheelEvent` param? I think either these should all be react events in the param, or `notifyMouseHandlers` should expect base browser events if they aren't expected to always be react events

Reply:
Yea some of these handlers are added directly on the window (handleMouseUp) or need to be added directly to get the `passive` option (handleWheel), so they are in fact expecting window events... Really it doesn't matter which kind of event it is, `MouseEvent | React.MouseEvent`, since the properties we check are on either.
I'll make a `type GridMouseEvent = MouseEvent | React.MouseEvent` so it shouldn't matter